server/src/main/java/io/seata/server/storage/file/session/FileSessionManager.java

```diff
@@ -120,10 +120,14 @@ public void removeGlobalSession(GlobalSession session) throws TransactionExcepti
     @Override
     public List<GlobalSession> findGlobalSessions(SessionCondition condition) {
         List<GlobalSession> found = new ArrayList<>();
-        for (GlobalSession globalSession : sessionMap.values()) {
-            if (System.currentTimeMillis() - globalSession.getBeginTime() > condition.getOverTimeAliveMills()) {
-                found.add(globalSession);
+        if (!condition.isLazyLoadBranch()) {
```

Comment:
LazyLoadBranch should be ignored in file storage mode.

Reply:
done